src/plugin/context/systemparams/rapier_context_access.rs

```diff
@@ -0,0 +1,125 @@
+use bevy::ecs::system::SystemParam;
+use bevy::prelude::*;
+use std::ops::{Deref, DerefMut};
+
+use super::super::{DefaultRapierContext, RapierContext, RapierContextEntityLink};
+/// Utility [`SystemParam`] to easily access the default world [`RapierContext`] immutably
+#[derive(SystemParam)]
+pub struct DefaultRapierContextAccess<'w, 's, T: Component = DefaultRapierContext> {
+    rapier_context: Query<'w, 's, &'static RapierContext, With<T>>,
+}
+
+impl<'w, 's, T: Component> DefaultRapierContextAccess<'w, 's, T> {
+    /// Use this method if you only have one world.
+    pub fn single(&'_ self) -> &RapierContext {
+        self.rapier_context.single()
+    }
+}
+
+impl<'w, 's> Deref for DefaultRapierContextAccess<'w, 's> {
+    type Target = RapierContext;
+
+    fn deref(&self) -> &Self::Target {
+        self.rapier_context.single()
+    }
+}
+
+/// Utility [`SystemParam`] to easily access the default world [`RapierContext`] mutably
```

Comment:
Same as above - it should be made clear that usage of this struct's methods/dereferencing it will cause a panic if the number of worlds that has this component is not 1.